hvplot/tests/testibis.py

```diff
@@ -0,0 +1,73 @@
+"""Ibis works with hvplot"""
+import sqlite3
+from pathlib import Path
+
+import holoviews as hv
+import matplotlib
+import pytest
+
+matplotlib.use('agg')
```

Comment:
Is this required?

Reply:
Yes. Otherwise it failed.

Reply:
Alright true, HoloViews test suite sets up an environment variable `MPLBACKEND` to `"Agg"` (https://github.com/holoviz/holoviews/blob/master/.github/workflows/test.yaml#L44).